unstructured/ingest/v2/cli/cmds/salesforce.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.v2.cli.base import SrcCmd
+from unstructured.ingest.v2.cli.interfaces import CliConfig
+from unstructured.ingest.v2.cli.utils import DelimitedString
+from unstructured.ingest.v2.processes.connectors.salesforce import CONNECTOR_TYPE
+
+
+@dataclass
+class SalesforceCliConnectionConfig(CliConfig):
+    @staticmethod
+    def get_cli_options() -> list[click.Option]:
+        options = [
+            click.Option(
+                ["--username"],
+                required=True,
+                type=str,
+                help="Salesforce username usually looks like an email.",
+            ),
+            click.Option(
+                ["--consumer-key"],
+                required=True,
+                type=str,
+                help="For the Salesforce JWT auth. Found in Consumer Details.",
+            ),
+            click.Option(
+                ["--private-key"],
+                required=True,
+                type=str,
+                help="Path to the private key or its contents for the Salesforce JWT auth. "
+                "Key file is usually named server.key.",
+            ),
+        ]
+        return options
+
+
+@dataclass
+class SalesforceCliIndexerConfig(CliConfig):
+    @staticmethod
+    def get_cli_options() -> list[click.Option]:
+        possible_categories = ["Account", "Case", "Campaign", "EmailMessage", "Lead"]
+        options = [
+            click.Option(
+                ["--categories"],
+                default=None,
+                required=True,
+                type=DelimitedString(choices=possible_categories),
```

Comment:
Can we import the categories straight from the connector file rather than redefine them here so there's one source of truth for that.